Xamarin.Forms.Platform.UAP/BoxViewRenderer.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System.ComponentModel;
+﻿using System;
+using System.ComponentModel;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Automation.Peers;
 using Windows.UI.Xaml.Shapes;
 
 namespace Xamarin.Forms.Platform.UWP
 {
+	[Obsolete("Replaced with BoxViewRenderer2")]
```

Comment:
Please have a look at how other obsolete messages look and make it more like that

Reply:
"BoxViewRenderer is obsolete as of version 4.4. Please use BoxViewRenderer2 instead."  I'm taking a guess at the version this might get merged into, I might put 4.x and update it once everything is approved.